Lib/urllib.py

```diff
@@ -203,7 +203,9 @@ def open(self, fullurl, data=None):
         name = 'open_' + urltype
         self.type = urltype
         name = name.replace('-', '_')
-        if not hasattr(self, name):
+
+        # bpo-35907: # disallow the file reading with the type not allowed
```

Comment:
I don't understand the # in "# bpo-35907: # disallow ...". Is it a typo?

Reply:
Oh Sorry, my mistake. I will correct it.

Reply:
I have fixed the typo.